### ISSUE TITLE
Contribution screen is gone when leaving tab

### ISSUE THREAD

**Issue by prk-dismoi** [dis-moi/mobile_poc#219]
### 1. Configuration

* Device brand `(Samsung, etc...)`: Samsung
* Device name `(Galaxy S8, etc...)`: Galaxy S8
* Device model number `(settings > about phone)`: SM - G950F
* Android OS version `(settings > about phone > Android version)`: 9
* Browser: **Chrome**
* Browser version `(settings > apps > Chrome > bottom of page)`: 91.0.4472.164
* Dismoi App version `(settings > apps > Dismoi > bottom of page)`: 1.0.7 (6)
---
### 2. Describe the current behavior:

If I open the contribution screen from a bubble, navigate away from Chrome while the contribution screen is showing (open other tab, or back to home) then back to the page where the contribution was showing, its gone.

---
### 3. Steps to reproduce:

    a. sign up to a contributor
    b. click on see example
    c. click on the bubble to open the contribution screen
    d. navigate away from Chrome/this tab (got to home, or open other tab)
    e. get back to the page: the contribution screen is gone.

---
### 4. Describe the expected behavior

Once opened, the contribution screen should remain available on the page it was triggered on, unless the page is closed.

---
### 4. Additional details (optional)

Write here...

**Comment by felix-lambert** [comment]
Hello @prk-dismoi this is the predicted behavior so not really a bug: I remove the contribution layout if you get out of chrome. Today, I estimate that leaving the page is like a deletion of the layout, so no more bubble or layout. If you download the page again, the bubble will appear, but if you come back to the page no more layout.

Are we sure this is really a problem? I personally think that this behavior is ok because we try not to bother the user too much. If we want the layout to show again when you go back to chrome, this is considered as a new feature for me. To discuss with @MaartenLMEM I guess.

**Comment by prk-dismoi** [comment]
good point. @MaartenLMEM any thoughts on this?

**Comment by MaartenLMEM** [comment]
Yes it s a new feature. There are other littles cases like these, I think it has to be designed together. 
Because there are many conditions that are involved in D notification and detailed view diplay : 
- of course followed, not followed
- readen (detailed view diplayed) ornot readen 
- closed, not closed
- trashed, not trashed
etc.
We will come back with answsers soon.
